analytics.go

```diff
@@ -311,7 +378,7 @@ func (c *client) upload(b []byte) error {
 func (c *client) report(res *http.Response) (err error) {
 	var body []byte
 
-	log.Println("Response %d %s", res.StatusCode, res.Status)
+	log.Printf("Response %d %s", res.StatusCode, res.Status)
```

Comment:
You might want to add a \n at the end.

Reply:
This line was unnecessary.So, it is removed